docs/cody/clients/install-jetbrains.mdx

```diff
@@ -97,17 +93,59 @@ In addition, Autocomplete suggestions use the same color as inline parameter hin
 
 ## Commands
 
-Cody with JetBrains offers quick, ready-to-use Commands for common actions to write, describe, fix, and smell code. These allow you to run predefined actions with smart context-fetching anywhere in the editor, like:
+Cody with JetBrains offers quick, ready-to-use Commands for common actions to write, describe, fix, and smell code. These allow you to run predefined actions with smart context-fetching anywhere in the editor.
+
+These are broadly categorized into the Edit Commands and Chat Commands.
+
+- **Edit Code** (Beta): Makes inline code edits. You also get the option to select LLM for edit suggestions
+- **Document Code** (Beta): Create inline documentation for your code
+- **Generate Unit Test** (Beta): Creates inline unit tests for your code-
```

Comment:
I think we can remove the Beta labels for Edit, Document, and Generate Unit Tests. We won't have a live version in the marketplace before GA (when these docs should go live).